src/components/Glp/GlpSwap.js

```diff
@@ -464,6 +479,23 @@ export default function GlpSwap(props) {
     return [false];
   };
 
+  const renderErrorModal = useCallback(() => {
+    const inputCurrency = swapToken.address;
+    const oneInchUrl = `https://app.1inch.io/#/${chainId}/swap/${inputCurrency}`;
+    const label = `${swapToken.symbol} Capacity Reached`;
+    return (
+      <Modal isVisible={!!modalError} setIsVisible={setModalError} label={label} className="Error-modal">
```

Comment:
Boolean(modalError) is more clear than !!modalError

Reply:
Regarding the new structure - could you please move this component to a new file? 

Maybe I didn't quite understand something, but now it looks like a regular component, why is it written as a callback?

Reply:
`Boolean()` is more explicit and makes sense. I will update that.

That's correct `useCallback` does not make sense here. Removed
I feel it makes sense to leave it where it is because it's small and very specific. Moving as a new component will make it look more complicated I feel. But please let me know your thoughts.

Reply:
Makes sense. I will updated it